### PR TITLE
chore(skip-release): bump sinceIdea to 2022.3

### PR DESCRIPTION
**Pull request by adietish**
https://github.com/redhat-developer/intellij-common/pull/219

### CODE REVIEW

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2022.1, IU-2022.2, IU-2022.3, IU-2023.1, IU-2023.2]
+        IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2024.1]
```
Comment (sbouchet) [inline .github/workflows/IJ.yml:17]: IU-2023.3 is missing

**Review by sbouchet — CHANGES_REQUESTED** [review]

File: .github/workflows/IJ.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IU-2022.1, IU-2022.2, IU-2022.3, IU-2023.1, IU-2023.2]
+        IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.2, IU-2024.1]
```
Comment (sbouchet) [inline .github/workflows/IJ.yml:17]:
```suggestion
        IJ: [IU-2022.3, IU-2023.1, IU-2023.2, IU-2023.3, IU-2024.1]
```

**Review by sbouchet — CHANGES_REQUESTED** [review]
still a typo

**Comment by adietish** [comment]
@sbouchet: ready to merge?

**Review by sbouchet — APPROVED** [review]
LGTM

**Pull request merged**